accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -0,0 +1,627 @@
+=====
+Chile
+=====
+
+Introduction
+============
+
+The Chilean localization has been improved and extended in Odoo v13,in this version the next 
+modules are available:
+
+- **l10n_cl:** Adds accounting features for the Chilean localization, which represent the minimal 
+  configuration required for a company to operate in Chile and under the SII 
+  (Servicio de Impuestos Internos) regulations and guidelines.
+  
+- **l10n_cl_edi:** Includes all technical and functional requirements to generate and receive 
+  Electronic Invoice via web service, based on the SII regulations. 
+
+- **l10n_cl_reports:** Adds the reports Propuesta F29 y  Balance Tributario (8 columnas). 
+
+Configuration
+=============
+
+Install the Chilean localization modules
+----------------------------------------
+For this, go to *Apps* and search for Chile. Then click *Install* in the module Chile 
+E-invoicing, this module has a dependency with Chile - Accounting in case this last
+one is not installed, Odoo installs it automatically with E-invoicing.
```

Comment:
```suggestion
For this, go to *Apps* and search for Chile. Then click *Install* in the module **Chile 
E-invoicing**. This module has a dependency with **Chile - Accounting**. In case this last
one is not installed, Odoo installs it automatically with E-invoicing.
```